Environments.Rmd

```diff
@@ -896,7 +898,7 @@ As well as powering scoping, environments are also useful data structures in the
 
 ## Quiz answers {#env-answers}
 
-1.  There are four ways: every object in an environment must have a name;
```

Comment:
Even if my alterations are not implemented, this must be changed to be consistent with #734